scripts/load-docker-compose.sh

```diff
@@ -82,7 +24,21 @@ else
     sleep 10
 fi
 
+# Add path "/scripts/" in front of each non-option argument.
+for opt do
+	case $opt in
```

Comment:
```suggestion
	case "$opt" in
```

Reply:
Quotes actually never needed here.

Reply:
(because `case` is a keyword, not a command, which is why you don't have to quote `$name` in `[[ -f $name ]]` either)